src/main/java/lotto/view/InputView.java

```diff
@@ -0,0 +1,43 @@
+package lotto.view;
+
+import java.util.Arrays;
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class InputView {
+    private static final String WINNING_NUMBERS_FORMAT = "^\\d{1,2}(,\\s\\d{1,2}){5}$";
+
+    private static final Scanner scanner = new Scanner(System.in);
```

Comment:
상수로 선언하셨으니 이름을 대문자로 작성해보면 어떨까요?
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```